schemas/CustomFieldCreate.yaml

```diff
@@ -0,0 +1,64 @@
+type: object
```

Comment:
discussion: what do you think about the next format `<EntityType>.<operation>.yaml`

- CustomField.create.yaml
- CustomField.update.yaml
- but: CustomFieldValue.yaml